src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -1202,11 +1202,14 @@ internal ImmutableArray<DocumentInfo> GetGeneratedDocuments(ProjectId projectId)
                 return ImmutableArray<DocumentInfo>.Empty;
             }
             var compilation = await this.CreateCompilationAsync(projectState, cancellationToken).ConfigureAwait(false);
+            ImmutableArray<Diagnostic> diagnostics;
```

Comment:
Good catch. Any diagnostics here should probably be reported as Intellisense errors. I'll log a bug.